src/server/google-cloud.md

```diff
@@ -15,7 +15,11 @@ For information about other Google APIs (including Firebase)
 that you might want to use from Dart code,
 see the [Google APIs page](/guides/google-apis).
 
-## Cloud Run
+## Recommended solutions
+
+To run Dart in the Cloud, we recommand using serverless computing solutions.
```

Comment:
```suggestion
To run Dart in the Cloud, we recommend using serverless computing solutions.
```

Reply:
I guess...if you care about..."spelling" 🤷  🙃